### PR TITLE
use rails activesupport native redis cache

### PR DESCRIPTION
**Pull request by mogman1**
The [redi-activesupport](https://github.com/redis-store/redis-activesupport) gem stopped being maintained in favor of [Rails' native support for Redis in ActiveSupport](https://github.com/rails/rails/blob/6-0-stable/activesupport/lib/active_support/cache/redis_cache_store.rb).

In their implementation, Redis commands are [wrapped in a failsafe method that catches all `Redis::BaseError`s](https://github.com/rails/rails/blob/6-0-stable/activesupport/lib/active_support/cache/redis_cache_store.rb#L477-L482).  So the option for ignoring particular errors is no longer relevant.  Plus the `raise_errors?` method and option were a `redis-activesupport` exclusive and do not appear in ActiveSupport's Redis cache implementation.  I left the specs testing that the specific error that was causing issues in as a regression guard.

### CODE REVIEW

**Review by davideroberts — APPROVED** [review]

File: redis-cluster-activesupport.gemspec [hunk 1]
```diff
@@ -19,8 +19,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "redis-activesupport"
-  spec.add_development_dependency "bundler", "~> 1.15"
+  spec.add_dependency "activesupport", "~> 6.0"
```
Comment (davideroberts) [inline redis-cluster-activesupport.gemspec:22]: What's the thinking with locking this to 6 vs >= 5.2? I believe you mentioned the method signatures were different. Do we want to try supporting 5.2?

File: spec/active_support/cache/redis_cluster_store_spec.rb [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 describe ::ActiveSupport::Cache::RedisClusterStore do
   let(:options) { {} }
+  let(:redis) { subject.redis }
 
   subject { described_class.new(options) }
```
Comment (davideroberts) [inline spec/active_support/cache/redis_cluster_store_spec.rb]: In order to support Ruby 3, could you please change the args to `*options`?

```
subject { described_class.new(*options) }
```

File: redis-cluster-activesupport.gemspec (hunk 1 above)
Comment (mogman1) [reply redis-cluster-activesupport.gemspec:22]: My thinking is that we won't have a chance to really vet this in Rails 7 yet, but will in Rails 6, so I wanted to be conservative in what's permitted.  The old `redis-activesupport` gem was supported up to 5.2, so 6.0 seemed like a reasonable floor to start from.

**Comment by mogman1** [comment]
It took a fair amount of trial-and-error, but I think I got [Github Workflows working to run the specs](https://github.com/mogman1/redis-cluster-activesupport/actions/runs/4128137542).  It should be testing for ruby 2.6 - 3.2 and jruby 9.3 and 9.4 with Rails 6.0 & 6.1.  I've never really used workflows before, so any feedback would be appreciated.

File: gemfiles/rails61.gemfile.lock [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    rspec-support (3.12.0)
+    spoon (0.0.6)
+      ffi
+    tzinfo (2.0.6)
+      concurrent-ruby (~> 1.0)
+    zeitwerk (2.6.6)
+
+PLATFORMS
+  universal-java-1.8
+  universal-java-11
```
Comment (mogman1) [inline gemfiles/rails61.gemfile.lock:56]: Github Workflows wanted this platform and I couldn't find a way to have it use java 8 instead.

**Comment by film42** [comment]
Looks great!

**Comment by film42** [comment]
Released a v1.0.0

**Pull request merged**